com.unity.shadergraph/Editor/Generation/Targets/Fullscreen/FullscreenData.cs

```diff
@@ -0,0 +1,189 @@
+using UnityEditor.ShaderGraph;
+using UnityEngine;
+using static UnityEditor.Rendering.BuiltIn.ShaderUtils;
+using UnityEditor.Rendering.BuiltIn;
+using System;
+using UnityEditor.ShaderGraph.Serialization;
+using UnityEngine.Rendering;
+using BlendMode = UnityEngine.Rendering.BlendMode;
+using BlendOp = UnityEditor.ShaderGraph.BlendOp;
+
+namespace UnityEditor.Rendering.Fullscreen.ShaderGraph
+{
+    internal class FullscreenData : JsonObject
```

Comment:
Why was this class done as a JsonObject? I dont see any refs that couldn't be serialized in a normal fashion.

Reply:
I need the `JsonObject` because the `m_SubTargetData` in the Target is a list of JsonData`<JsonObject>`. `JsonData` only accepts a `JsonObject` as a template, so unless I remove completely the `JsonData` and use something else, I don't see how I can get rid of `JsonObject` on the `FullscreenData`.

The only alternative I know regarding polymorphic serialization besides JSON is [SerializeReference] but I don't think it works in ShaderGraph.